docs/build/guides/cli/payments-and-assets.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Payments and Assets
+hide_table_of_contents: true
+description: Send XLM, stellar classic, or a soroban asset using the Stellar CLI.
+---
+
+To send payments and work with assets using the Stellar CLI, follow these steps:
+
+1. Fund the accounts:
```

Comment:
WDYT about `export STELLAR_NETWORK=testnet` as the first step in the guide? Would shorten commands a bit